packages/nimble-components/src/select/specs/IxD.md

```diff
@@ -188,7 +197,9 @@ When the dropdown is open, clicking anywhere outside of a list option (including
 
 ![ ](spec-images/select-23.png)
 
-> NOTE: NEEDS VISUAL DESIGN FOR MULTIPLE FOCUS STATES?
+If the select component allows clearing the selected item, the _Clear selection button_ will appear when an option is selected. Clicking the button will clear the selected item and revert to the placeholder state.
```

Comment:
One thing that needs clarification is where the clear button should be placed relative to the error visible icon (when it is visible). I presume this is something that @NIbokeefe will specify?

Reply:
![image](https://github.com/ni/nimble/assets/1458528/0a2591c7-d590-40b2-968f-2c20d5ecfd18)?

Yes, I'll ask Brandon to do a visual pass for the clear button, the various clear button states, and the error icon placement.

Reply:
@fredvisser @atmgrifter00 @jattasNI 

In the select control section on [https://www.figma.com/file/PO9mFOu5BCl8aJvFchEeuN/Nimble_Components?type=design&node-id=9012%3A327273&mode=design&t=l9M1XTBQ0yvFnNny-1](url) Figma I've drawn some workflows and design options.

<img width="1132" alt="Screenshot 2024-04-10 at 4 37 42 PM" src="https://github.com/ni/nimble/assets/106594349/3b510f5a-0807-4ef5-b7c0-16b555fa299d">

Reply:
@NIbokeefe I'm guessing this was already your plan but in case not: let's get the team's feedback at our meeting today.